site/docs/components/drawer/usage.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Drawer
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+Use the Drawer component by wrapping it around your content and setting the desired position. Provide a mechanism, such as a Button, for users to open and close it.
+
+When open, the Drawer displays a visual overlay, also known as a scrim, over the underlying content. It also plays relevant animations when opening and closing that can be visually customized via CSS.
+
+### When to use Drawer
+
+- Use Drawer when you need additional content, such as a form or sidebar navigation, to be displayed in a specific area of the screen.
+
+### When not to use Drawer
+
+- If you want your content split between pages or tabs, consider [Deck Layout](../deck-layout) instead.
+
+### Import
+
+To import Drawer from the lab Salt package, use:
+
+```js
+import { Drawer, useDrawer } from "@salt-ds/lab";
```

Comment:
Might be useful to add some explanation around `useDrawer`

Reply:
sounds good. I'm not sure exactly what to write, maybe something like (but more detailed):

Use the `useDrawer` hook to handle interactions with the Drawer, such as opening and closing it.

Reply:
Sounds good! Maybe a quick code snippet would help as well 😄

Reply:
I guess it would be the same code as in the examples. I'll just add the sentence for now.